### PR TITLE
Fix lib and viewer on linux-y systems.

### PR DESCRIPTION
**Pull request by mcarlen**
This fixes the library (lib/ includes/), the binaries in tools/ and the viewer in inventor/

### CODE REVIEW

File: Makefile.config [hunk 1]
```diff
@@ -14,7 +14,7 @@ LIB_INC=${PIXIEHOME}/lib
 LIB+=-lri
 else
 # Compilation Flags (also used for the linking)
-CXXFLAGS=-g -O2 -Wall -Werror #-Wno-non-template-friend
+CXXFLAGS=-g -O2 -fno-use-cxa-atexit -Wall -Werror #-Wno-non-template-friend
```
Comment (benlaurie) [inline Makefile.config:17]: I am curious why no-use-cxa-atexit is needed...

**Comment by benlaurie** [comment]
LGTM.

File: Makefile.config (hunk 1 above)
Comment (mcarlen) [reply Makefile.config:17]: I must have done something awful with global static variables, the linker error
I'm getting is this [1].

[1]
ld: TubeBundle_float.o: in function `__static_initialization_and_destruction_0':
/usr/include/c++/8/iostream:74: undefined reference to `__dso_handle'
ld: TubeBundle_float.o: relocation R_X86_64_PC32 against undefined hidden symbol `__dso_handle' can not be used when making a shared object
ld: final link failed: bad value

**Comment by benlaurie** [comment]
Haha - I may build it on Linux to see if I can fix this at some point.

**Pull request merged**